src/language/branches.md

```diff
@@ -129,20 +129,22 @@ check out the patterns documentation on [Switch statements and expressions][].
 
 ### Switch expressions
 
-A `switch` expression is similar to a `switch` statement, but you can use them
-anywhere you can use an expression. They produce a value based on the expression
-body of whichever case matches.
+A `switch` _expression_ produces a value based on the expression
+body of whichever case matches. 
+You can use a switch expression wherever Dart allows expressions*, for example:
 
-The syntax of a `switch` expression differs from `switch` statement syntax:
+```dart
+var x = switch (y) { ... };
 
-- Cases _do not_ start with the `case` keyword.
-- A case body is a single expression instead of a series of statements.
-- Each case must have a body; there is no implicit fallthrough for empty cases.
-- Case patterns are separated from their bodies using `=>` instead of `:`.
-- Cases are separated by `,` (and an optional trailing `,` is allowed).
-- Default cases can _only_ use `_`, instead of allowing both `default` and `_`.
+print(switch (x) { ... };
+
+return switch (x) { ... };
+```
 
-Switch expressions allow you to rewrite a _statement_ like this:
+*You _can't_ use a switch expression at the start of an expression statement.
```

Comment:
How about this one?

```suggestion
You _can't_ use a switch expression at
 the start of an expression statement.
```

Reply:
Yeah that was the caveat, like before it said "this is possible..." and this is like "..._unless_...". Again, if that's confusing to you I might be wrongly assuming that normally expressions _can_ be used at the start of an expression statement, just not _switch_ expressions.